app/views/admin/categories/destroy.js.erb

```diff
@@ -0,0 +1,2 @@
+$("#categories").html("<%= j render @categories %>");
```

Comment:
destroy thì chỉ xóa element bị xóa thôi chứ e
chứ render lại toàn bộ làm gì

Reply:
Vâng ạ

Reply:
Nếu em xóa 1 cái elem html đi thì tự nhiên trang nó đang có 5 phần tử nó chỉ còn 4 phần tử thôi chị ơi. Trông nó cứ như nào ý chị ạ.

Reply:
@dailx-1056 thế nên e cần làm cho nó đủ 5 phần tử (get record tiếp theo và render trong destroy.js.erb) thay vì render lại và về page 1

giữa việc đọc sách có bookmark và kh có bookmark, thì hẳn ng ta sẽ chọn bookmark vì kh phải tìm lại vị trí mình vừa dừng chân nhỉ

Reply:
Dạ vâng ạ .